themes/cardinal_service/pdb_components/opportunities_list/src/Components/Filters.jsx

```diff
@@ -1,13 +1,61 @@
-import React, {Component} from 'react';
-import {SelectList} from "./SelectList";
-import {Slugs} from "./Slugs";
+import React, { Component } from 'react';
+import { SelectList } from './SelectList';
+import { Slugs } from './Slugs';
+import styled from 'styled-components';
 
 const _ = require('lodash');
 const lodashUuid = require('lodash-uuid');
 const queryString = require('query-string');
 
-export class Filters extends Component {
+const FilterContainer = styled.div`
+  margin: 20px 0;
+
+  .flex-2-of-12 {
+    display: flex;
+    justify-content: center;
+  }
+
+  @media (max-width: 768px) {
+    .flex-10-of-12,
+    .flex-2-of-12 {
+      flex: unset;
+      width: 100%;
+      max-width: unset;
+    }
```

Comment:
if you're changing their widths, you can use the classes `flex-md-10-of-12` or similar. available classes contain `sm`, `md`, `lg`, `xl` and `2xl`. see https://decanter.stanford.edu/page/layouts-grid-system/

Reply:
This is exactly what I needed! Thanks Mike — I'll update the classes